### DISCUSSION TITLE
When will Chia official pooling software be released?

### DISCUSSION THREAD

**Discussion by lhcdims** [Chia-Network/chia-blockchain#6253] [Q&A]
When will Chia official pooling software be released?

**Comment by lvcivs** [comment]
A fixed date has not been announced yet. If you are interested, you can watch a livestream on Wednesday June 2nd: https://github.com/Chia-Network/chia-blockchain/wiki/Pooling-FAQ#when-is-the-next-zoom-technical-qa-on-pooling

The team will likely provide more detailed information about the rollout of the pooling functionality during that livestream.
